backend/qemu.pm

```diff
@@ -563,7 +563,9 @@ sub start_qemu {
     {
         push(@params, @vgaoptions);
 
-        gen_params @params, "global", "isa-fdc.driveA=" unless $vars->{QEMU_NO_FDC_SET};
+        unless ($vars->{ARCH} eq 'aarch64' || $vars->{ARCH} eq 'arm' || $vars->{QEMU_NO_FDC_SET}) {
```

Comment:
Please add a comment describing why the parameter does not apply for aarch64 (or arm)